lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -3,24 +3,58 @@ module Compute
     class Ovirt
       class V4
         class Real
+          DEFAULT_PROVISIONED_SIZE = 1 * 2**30
+          DISK_INTERFACES = {
```

Comment:
The SDK will accept the string as is or might have a translation method, this should not be here.